.github/RELEASE.md

```diff
@@ -0,0 +1,85 @@
+# Operator Hub Release steps
+
+Set some environment variables used during the process
+
+```shell
+export new_version=<new-version>
+export old_version=<old-version>
+export quay_test_repo=<quay-test-repo>
+export community_fork=<a-fork-of-community-operator>
+```
+
+An example could be:-
+
+```shell
+export new_version=0.1.0
+export old_version=0.1.1
```

Comment:
I think these are backwards. Should the "new version" be 0.1.1, and the "old version" be 0.1.0?